cluster-autoscaler/core/scale_down.go

```diff
@@ -44,6 +44,8 @@ import (
 
 	"github.com/golang/glog"
 	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/autoscaler/cluster-autoscaler/utils/gpu"
+	"k8s.io/kubernetes/bazel-kubernetes/pkg/kubelet/kubeletconfig/util/log"
```

Comment:
We use glog literally everywhere. Please don't introduce different logging packages.